tests/commons/web/test_conneg.py

```
@@ -256,10 +256,10 @@ def test_conneg_eval(url, expected):
             else:
                 assert (
                     filename is not None
-                ), f"Missing filename for {url} {exp_suffix=}"
+                ), f"Missing filename for {url},{exp_mime},{exp_profile} {exp_suffix=}"
                 assert filename.endswith(
                     exp_suffix
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)